pkg/security/ptracer/fim_handlers.go

```diff
@@ -250,6 +251,11 @@ func registerFIMHandlers(handlers map[int]syscallHandler) []string {
 			ShouldSend: isAcceptedRetval,
 			RetFunc:    nil,
 		},
+		{
+			ID:      syscallID{ID: Pipe2Nr, Name: "pipe2"},
```

Comment:
shouldn't we handle `pipe` as well ? (even if not present on arm64)

Reply:
👍